<?xml version="1.0"?>

<launch>
    <arg name="swing_id" default="1"/>
    <arg name="target_dx" default="0.0"/>
    <arg name="target_dy" default="0.0"/>
    <arg name="target_dz" default="0.1"/>
    <arg name="swing_t" default="[0.5, 2.5]"/>
    <arg name="contact_det" default="False"/>


    <node name="casannis" pkg="casannis-walking" type="node1.py" output="screen">
        <param name="sw_id" value="$(arg swing_id)"/>
        <param name="tgt_dx" value="$(arg target_dx)"/>
        <param name="tgt_dy" value="$(arg target_dy)"/>
        <param name="tgt_dz" value="$(arg target_dz)"/>
        <param name="sw_t" value="$(arg swing_t)"/>
        <param name="cont_det" value="$(arg contact_det)"/>
    </node>
</launch>
